src/phoenix/server/api/input_types/CreateTraceAnnotationsInput.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+import strawberry
+from strawberry.relay import GlobalID
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class CreateTraceAnnotationsInput:
+    trace_id: GlobalID
+    name: str
+    annotator_kind: str
+    label: Optional[str] = None
+    score: Optional[float] = None
+    explanation: Optional[str] = None
+    metadata: JSON = dict()
```

Comment:
```suggestion
    metadata: JSON = strawberry.field(default_factory=dict)
```

If we instantiate the dictionary on the type, my expectation is that the dictionary will get reused across different invocations of the resolver (not 100% sure)

Reply:
Can also use `UNSET`.